optimum/onnx/graph_transformations.py

```diff
@@ -147,20 +147,32 @@ def merge_decoders(
 
     deduplicated_initializers = _deduplicated_cross_model_initializers([decoder, decoder_with_past], suffix=graph_name)
 
+    # keep initializers of dim 0 (or dim 1 + int32/int64) in subgraphs for readability purposes, and also because
```

Comment:
```suggestion
    # Keep initializers of dim 0 (or dim 1 + int32/int64) in subgraphs for readability purposes, and also because
```

- https://google.github.io/styleguide/pyguide.html#386-punctuation-spelling-and-grammar
- https://peps.python.org/pep-0008/#comments

Reply:
thanks